core/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -80,7 +80,7 @@
     private final boolean dynamic;
 
     private volatile String defaultMappingSource;
-    private volatile String defaultPercolatorMappingSource;
```

Comment:
Because we allow the default mapping source by using the special `_default_` type, while the default percolator source can't be changed.